Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo ()
+        {
+            string userFirstName = UserName(message: "Enter your first name: ");
+            string userSurname = UserName(message: "Enter your surname: ");
+            int userAge = UserAge(message: "Enter your age: ");
+            float userHeight = UserHeightWeight(message: "Enter your height in cm: ");
+            float userWeight = UserHeightWeight(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBMI(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBMI(float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4863638)

Reply:
Updated method name.